test/expected/sql_query_results_optimized.out

```diff
@@ -293,15 +293,15 @@ FROM hyper_1 GROUP BY t ORDER BY t DESC limit 2;
                      ->  Index Scan using _hyper_1_1_chunk_time_plain on _hyper_1_1_chunk
 (8 rows)
 
-SELECT time_bucket('1 minute', time - INTERVAL '30 seconds') t, avg(series_0), min(series_1), avg(series_2) 
+SELECT time_bucket('1 minute', time - INTERVAL '30 seconds') t, trunc(avg(series_0)::numeric, 8) as avg_trunc1, min(series_1), trunc(avg(series_2)::numeric, 8) as avg_trunc2
```

Comment:
Actually, can't we just set `extra_float_digits` instead of calling `trunc` on every float-like type? https://www.postgresql.org/docs/9.3/static/runtime-config-client.html#GUC-EXTRA-FLOAT-DIGITS

Reply:
Or, doesn't that work for numerics?

Reply:
Actually, nevermind. This setting seems to do something different.

Reply:
Playing around with the setting, it seems you can set it negatively to reduce the number of digits from the default `FLT_DIG`. However, if `FLT_DIG` is different between platforms it might not help anyway.

Reply:
So should I go with this approach or not?

Reply:
It doesn't appear to work with numerics